### PR TITLE
Integrate apache for production deployment

### PR DESCRIPTION
**Pull request by b-rohit**
Related to https://github.com/TrustPoint-Project/trustpoint-roadmap/issues/25

**Description of changes**
Changes in Dockerfile to generate ssl certificates, enable Apache
Apache configuration file

**Notes** 

**Legal** 
- [ ] I certify that I have all necessary rights to publish this contribution under the MIT license. I agree to the Trustpoint CLA and have added my name to the AUTHORS.md file.

### CODE REVIEW

**Comment by Aircoookie** [comment]
Great work!
As noted in the weekly:
- running migrations and creating superuser is not required to do manually, as it is taken care of by `reset_db`
- let's use 443 as default server port

File: Dockerfile [hunk 1]
```diff
@@ -1,32 +1,59 @@
...
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
 
-# Set the working directory in the container
-WORKDIR /app
+# Make port 80 and 443 available to the world outside this container.
+# 80 will be redirected to 443 using TLS through the apache.
+EXPOSE 80 443
+
+RUN apt-get update && apt-get install -y apt-utils vim curl apache2 apache2-utils python3 libapache2-mod-wsgi-py3 python3-pip python3-venv
```
Comment (BytesWelder) [inline Dockerfile:13]: I would remove unnecessary packages like vim (maybe curl as well? IDK), so the image stays slim and therefor minimize potential security vulnerabilities.

File: Dockerfile [hunk 2]
```diff
@@ -1,32 +1,59 @@
...
 ENV POETRY_HOME=/opt/poetry
 RUN python -m venv $POETRY_HOME && $POETRY_HOME/bin/pip install poetry==1.8.2
 
-# Copy the current directory contents into the container at /app
-COPY . /app
+# Copy the current directory contents into the container
+COPY ./ /var/www/html/trustpoint/
+
+# Set the working directory in the container
+WORKDIR /var/www/html/trustpoint/
```
Comment (BytesWelder) [inline Dockerfile:28]: Maybe for the beta it is not that important but bevor our first realease we should think about data persistency.
An outsourced db or at least a volume in a docker context so we do not lose our settings/data if the container dies or we restart it.

Comment (AlexHx8472) [reply Dockerfile:28]: can't we just make the sqlite3 file persistent for now?

**Pull request merged**